src/pages/register/index.js

```diff
@@ -0,0 +1,3 @@
+import Register from './register.component';
```

Comment:
Please, rename the file `register.component.js` => `register`

Reply:
renamed